app/components/login_button_component.rb

```diff
@@ -0,0 +1,26 @@
+class LoginButtonComponent < ButtonComponent
+  attr_reader :action, :color, :logo_path, :tag_options
```

Comment:
- `logo_path` is defined as a method, so I wouldn't expect it also as an `attr_reader` entry
- I'm not sure `action` is used?

```suggestion
  attr_reader :color, :tag_options
```